sdk/servicebus/Azure.Messaging.ServiceBus/src/ServiceBusConnection.cs

```diff
@@ -214,7 +214,7 @@ internal class ServiceBusConnection : IAsyncDisposable
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override string ToString() => base.ToString();
 
-        internal TransportSender CreateTransportSender(string entityName, ServiceBusRetryPolicy retryPolicy) =>
+        internal virtual TransportSender CreateTransportSender(string entityName, ServiceBusRetryPolicy retryPolicy) =>
```

Comment:
Given that `CreateTransportReceiver` was already marked `virtual`, I figured this wouldn't be a controversial change.